docs/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reacm-cm-ui-docs",
```

Comment:
We should fix this typo (in parent `package.json` file of course): `react-cm-ui-docs`
First time I've noticed that! :-)
Definitely a NAB for this PR ... it's been broken a while now.

Reply:
Fixed

Reply:
I fixed it in `/docs/package.json` in b3a5970 =)